### PR TITLE
remove commands before replacement by gui commands

### PR DESCRIPTION
**Pull request by yenda**
Tested: no more text commands on Android

### CODE REVIEW

**Comment by flexsurfer** [comment]
RIP cli ☠️

**Review by flexsurfer — APPROVED** [review]

**Comment by flexsurfer** [comment]
image

File: src/status_im/constants.cljs [hunk 1]
```diff
@@ -10,8 +10,6 @@
 (def content-type-text "text/plain")
 (def content-type-sticker "sticker")
 (def content-type-status "status")
-(def content-type-command "command")
```
Comment (flexsurfer) [inline src/status_im/constants.cljs:13]: @cammellos i've seen you make changes here as well. jfyi

Comment (cammellos) [reply src/status_im/constants.cljs:13]: yes, thanks, @yenda mentioned this as well

**Review by cammellos — APPROVED** [review]

**Comment by yenda** [comment]
@churik I think these errors are due to the fact that the account ran out of cash and not the PR itself?

**Comment by churik** [comment]
@yenda most of these failures are fixed, I rerun e2e.

**Comment by churik** [comment]
ah, can you please rebase your PR to current develop?

**Pull request merged**